### PR TITLE
fix(PRISMA_CLI_BINARY_TARGETS): Clarify

### LINKED ISSUE #6830: inconsistency between `PRISMA_CLI_BINARY_TARGETS` and `binaryTargets`
## Bug description

`native` is not allowed in **PRISMA_CLI_BINARY_TARGETS** env (it's a valid value in `binaryTargets`)

## How to reproduce

```
PRISMA_CLI_BINARY_TARGETS=native,linux-musl yarn prisma generate
```

it will shows:
```
Error: Unknown binaryTarget native and no custom binaries were provided
```

## Expected behavior

should works like `binaryTargets`?

## Environment & setup

- OS: Mac OS
- Database: MySQL
- Node.js version: 14.16.0
- Prisma version: 2.21.2 (e421996c87d5f3c8f7eeadd502d4ad402c89464d)

### PR DESCRIPTION
**Pull request by janpio**
closes https://github.com/prisma/prisma/issues/6830

### CODE REVIEW

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 1]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not work affect runtime of Prisma CLI or Prisma Client at all).
```
Comment (tanberry) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx:194]: @janpio for the last bit of the sentence... is it meant to say:  "... does not affect the runtime work of Prisma CLI or Prisma Client..."?

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 2]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
...
 
-For example:
-If you are on macOS, the binary target is `darwin` & AWS Lambda target is `rhel-openssl-1.0.x`
+This is to be used if you deploy to a specific platform via an upload of a local project including dependencies and your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`) and makes sure the target engine files are also downloaded.
```
Comment (tanberry) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx:196]: Consider breaking into two sentences: 
"Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded."

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 3]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
...
 
 ```terminal
 PRISMA_CLI_BINARY_TARGETS=darwin,rhel-openssl-1.0.x npm install
 ```
 
-This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets) which enables you to define the same setting for Prisma Client
+This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets) which enables you to define the same setting for Prisma Client.
```
Comment (tanberry) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]: "which" vs "that" issue... we can add a comma and say "... equivalent for the `binaryTargets` property of the `generator` block, which..." or we can use "that" ("... equivalent for the `binaryTargets` property of the `generator` block that enables you..."). In this sentence I prefer "which" with a comma. :-)

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx (hunk 1 above)
Comment (janpio) [reply content/400-reference/200-api-reference/300-environment-variables-reference.mdx:194]: "work" is just superfluous there.

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 4]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not work affect runtime of Prisma CLI or Prisma Client at all).
```
Comment (janpio) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client at all).
```

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx (hunk 1 above)
Comment (janpio) [reply content/400-reference/200-api-reference/300-environment-variables-reference.mdx:194]: Removed

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 5]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client at all).
 
-For example:
-If you are on macOS, the binary target is `darwin` & AWS Lambda target is `rhel-openssl-1.0.x`
+This is to be used if you deploy to a specific platform via an upload of a local project including dependencies and your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`) and makes sure the target engine files are also downloaded.
```
Comment (janpio) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
```

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx (hunk 3 above)
Comment (janpio) [reply content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets), which enables you to define the same setting for Prisma Client.
```

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 6]
```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client at all).
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client).
```
Feels better without

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**